demos/interactive_face_detection_demo_gapi/interactive_face_detection_gapi.hpp

```diff
@@ -0,0 +1,112 @@
+"(the list of available devices is shown below). Default value is CPU. The demo will look for a suitable plugin for device specified.";
+static const char thresh_output_message[] = "Optional. Probability threshold for detections";
+static const char bb_enlarge_coef_output_message[] = "Optional. Coefficient to enlarge/reduce the size of the bounding box around the detected face";
+static const char raw_output_message[] = "Optional. Output inference results as raw values";
+static const char no_wait_for_keypress_message[] = "Optional. Do not wait for key press in the end.";
+static const char no_show_processed_video[] = "Optional. Do not show processed video.";
+static const char dx_coef_output_message[] = "Optional. Coefficient to shift the bounding box around the detected face along the Ox axis";
+static const char dy_coef_output_message[] = "Optional. Coefficient to shift the bounding box around the detected face along the Oy axis";
+// TODO: Make this option valid for single image case
+static const char loop_video_output_message[] = "Optional. Enable playing video on a loop";
+static const char no_smooth_output_message[] = "Optional. Do not smooth person attributes";
+static const char no_show_emotion_bar_message[] = "Optional. Do not show emotion bar";
+static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
+
+// TODO: Support options:
+// Set number of maximum simultaneously processed faces for Age/Gender,
+// Head Pose Estimation, Emotions Recognition, Facial Landmarks Estimation networks
+
+// Enabling dynamic batch size for Age/Gender, Head Pose Estimation, Emotions Recognition,
+// Facial Landmarks Estimation networks
+
+// Enabling per-layer performance report
+
+// Enabling using of CPU custom layers.
+// Enabling using of GPU custom kernels.
+
+// Set maximum FPS for playing video
+
+DEFINE_bool(h, false, help_message);
+DEFINE_string(i, "", input_video_message);
+DEFINE_string(o, "", output_video_message);
+DEFINE_string(m, "", face_detection_model_message);
+DEFINE_string(m_ag, "", age_gender_model_message);
+DEFINE_string(m_hp, "", head_pose_model_message);
+DEFINE_string(m_em, "", emotions_model_message);
+DEFINE_string(m_lm, "", facial_landmarks_model_message);
+DEFINE_string(d, "CPU", target_device_message);
+DEFINE_string(d_ag, "CPU", target_device_message_ag);
+DEFINE_string(d_hp, "CPU", target_device_message_hp);
+DEFINE_string(d_em, "CPU", target_device_message_em);
+DEFINE_string(d_lm, "CPU", target_device_message_lm);
+DEFINE_bool(r, false, raw_output_message);
+DEFINE_double(t, 0.5, thresh_output_message);
+DEFINE_double(bb_enlarge_coef, 1.2, bb_enlarge_coef_output_message);
+DEFINE_bool(no_wait, false, no_wait_for_keypress_message);
+DEFINE_bool(no_show, false, no_show_processed_video);
+DEFINE_double(dx_coef, 1, dx_coef_output_message);
+DEFINE_double(dy_coef, 1, dy_coef_output_message);
+DEFINE_bool(loop_video, false, loop_video_output_message);
```

Comment:
We have settled on `-loop` as the name for flags like these, so please rename it.

Reply:
Done